### PR TITLE
🚀  Release v1.7.4

### PR DESCRIPTION
**Pull request by vishalanarase**
/area release

### CODE REVIEW

File: CHANGELOG/v1.7.4.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.3
```
Comment (sbueringer) [inline CHANGELOG/v1.7.4.md:8]: Let's please add: https://github.com/kubernetes-sigs/cluster-api/pull/10842

**Comment by sbueringer** [comment]
/lgtm

so far. We might have 1-2 more PRs until the release tomorrow. I'll comment accordingly

File: CHANGELOG/v1.7.4.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## :chart_with_upwards_trend: Overview
+- 22 new commits merged
+- 5 feature additions ✨
+- 5 bugs fixed 🐛
+
+## :sparkles: New Features
+- CABPK: Add kubeadm v1beta4 types (#10709)
```
Comment (sbueringer) [inline CHANGELOG/v1.7.4.md:15]:
```suggestion
- CABPK: Add kubeadm v1beta4 types (#10771)
```

(that's my fault, should have dropped the link from the commit: https://github.com/kubernetes-sigs/cluster-api/pull/10771/commits)

File: CHANGELOG/v1.7.4.md (hunk 1 above)
Comment (sbueringer) [reply CHANGELOG/v1.7.4.md:8]: Let's also add https://github.com/kubernetes-sigs/cluster-api/pull/10850

Comment (vishalanarase) [reply CHANGELOG/v1.7.4.md:8]: Added

File: CHANGELOG/v1.7.4.md (hunk 2 above)
Comment (vishalanarase) [reply CHANGELOG/v1.7.4.md]: Done

File: CHANGELOG/v1.7.4.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.3
+## :chart_with_upwards_trend: Overview
+- 23 new commits merged
+- 6 feature additions ✨
+- 5 bugs fixed 🐛
+
+## :sparkles: New Features
+- Bootstrap: Allow CAPBK to generate JoinConfiguration discovery kubeconfig (#10842)
```
Comment (sbueringer) [inline CHANGELOG/v1.7.4.md:15]: @vishalanarase I think you missed the second one I commented: https://github.com/kubernetes-sigs/cluster-api/pull/10845#discussion_r1669701391

Comment (vishalanarase) [reply CHANGELOG/v1.7.4.md:15]: Added

File: CHANGELOG/v1.7.4.md
Comment (adilGhaffarDev) [inline CHANGELOG/v1.7.4.md:1]: /lgtm
if we don't get any new PRs this should be good to go in.
@Sunnatillo @chandankumar4 please take a look

**Review by chandankumar4 — APPROVED** [review]
/lgtm

File: CHANGELOG/v1.7.4.md (hunk 3 above)
Comment (sbueringer) [reply CHANGELOG/v1.7.4.md:15]: Thank you!

**Comment by sbueringer** [comment]
Thx!!

/lgtm
/approve

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Pull request merged**